lib/pbench/cli/server/shell.py

```diff
@@ -36,11 +36,11 @@ def main():
         [
             "gunicorn",
             "--workers",
-            str(workers),
+            workers,
             "--pid",
             "/run/pbench-server/gunicorn.pid",
             "--bind",
-            f"{str(host)}:{str(port)}",
-            f"pbench.cli.server.shell:app())",
+            f"{host}:{port}",
+            "pbench.cli.server.shell:app())",
```

Comment:
A local `tox -e lint` failed for me due to `flake8` catching the empty f-string.  Not sure how this made it past all the commit checks.  Made this a separate commit that we could take in another PR.